src/lib/y2storage/planned/md.rb

```diff
@@ -94,6 +96,20 @@ def add_devices(md_device, devices)
       def self.to_string_attrs
         [:mount_point, :reuse, :name, :lvm_volume_group_name, :subvolumes]
       end
+
+    protected
+
+      # Values for volume specification matching
+      #
+      # @see MatchVolumeSpec
+      def volume_match_values
+        {
+          mount_point:  mount_point,
+          size:         DiskSize.zero,
```

Comment:
Why zero?

Reply:
Probably it does not make sense. The problem is that Planned::Md has not a size and I set it to zero as the equivalent nil value for a DiskSize. I will change it to actually use nil and it will be taken into account when matching size.